### ISSUE TITLE
How to do clustering with the output of this tool?

### ISSUE THREAD

**Issue by Mark-wt** [ZhangLabGT/scDisInFact#5]
Hello, thanks for very useful tool for batch correction. I can use the output (adata.obs['X_harmony']) of harmony to do neighboring graph and clustering based on leiden, so i wonder how i can do the next clustering (with leiden) using the output of your tool. Is it straightforward to use shared-bio factors as latent space features to construct the neighborhood graph? If so, shared-bio factors with a dimension of 8 is not enough, what is the maximum dimension that could go up to, for example 50 would be ok?

**Comment by PeterZZQ** [comment]
Yes, the shared-bio factors should be used for cell type identification. 

8 is enough for Leiden clustering or neighborhood graph construction. The procedure is all the same as other dimension reduction methods like Seurat or PCA. You can just directly run the neighborhood construction methods (e.g. in sklearn) or Leiden clustering algorithm (e.g. in scanpy) with x representation to be the shared-bio factor z.

**Comment by Mark-wt** [comment]
> Yes, the shared-bio factors should be used for cell type identification.
> 
> 8 is enough for Leiden clustering or neighborhood graph construction. The procedure is all the same as other dimension reduction methods like Seurat or PCA. You can just directly run the neighborhood construction methods (e.g. in sklearn) or Leiden clustering algorithm (e.g. in scanpy) with x representation to be the shared-bio factor z.

Ok, thanks for your replying, and i will try.
